src/containers/my-quizzes/create-or-edit-quiz-question/CreateOrEditQuizQuestion.tsx

```diff
@@ -26,20 +25,26 @@ import {
   UpdateQuestionParams
 } from '~/types'
 import { initialValues } from '~/containers/my-quizzes/create-or-edit-quiz-question/CreateOrEditQuizQuestion.constants'
+interface openModalProps {
```

Comment:
Please start interface's name with a capital letter

Reply:
not relevant